WPMedia/fusion-news-theme-blocks
# WIP: removes scss font override

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
Just removed a scss override.

## Jira Ticket
- [TMEDIA-270](https://arcpublishing.atlassian.net/browse/TMEDIA-270)

## Acceptance Criteria
Remove instances of 
```
   "sassVariableOverrides": [
   "$primary-font-family: $theme-primary-font-family;",
   "$secondary-font-family: $theme-primary-font-family;",
   "$primary-color: $theme-primary-color;"
   ],
```

## Test Steps
1. Checkout this branch `git checkout TMEDIA-270/remove_scss_overrides`
2. Run fusion repo with linked blocks `npx fusion start -l @wpmedia/header-nav-chain-block`
3. Check that the `header-nav-chain-block` links here: https://corecomponents.arcpublishing.com/pf/all-blocks-jc/?_website=the-gazette
4. Observe no styles have changed here: http://localhost/pf/allblockspr/?_website=the-gazette


## Effect Of Changes
Just a code cleanup, everything should look/operate the same

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
